packages/components/organisms/f-checkout/src/tenants/en-GB.js

```diff
@@ -43,59 +43,59 @@ const messages = {
         errorHeading: 'Error',
         genericServerError: 'Something went wrong, please try again later',
         pageLoad: {
-            heading: 'We can\'t show you this page',
-            description: 'It\'s a problem at our end, sorry. Your basket is safe and sound, though, so try again soon.'
+            heading: 'We can’t show you this page',
+            description: 'It’s a problem at our end, sorry. Your basket is safe and sound, though, so try again soon.'
         },
 
         guestUserCreationFailure: 'Guest checkout isn’t available, sorry. Try again soon or sign up',
 
         checkoutIssues: {
             MINIMUM_ORDER_VALUE_NOT_MET: {
-                title: 'Your order does not meet the minimum spend required',
-                message: 'The restaurant might have run out or updated its menu while you were ordering, sorry. You can go back to check your order and choose another item.',
-                buttonText: 'Back to Order'
+                title: 'Minimum spend not reached',
+                message: 'We’re sorry, but the minimum spend for your order has not been reached. Please amend your order.',
+                buttonText: 'Back to order'
             },
 
             RESTAURANT_NOT_TAKING_ORDERS: {
-                title: 'We couldn\'t send your order',
-                message: 'The restaurant went offline, so we couldn\'t send your order to them, sorry. Hopefully it\'s only temporary, please try again.',
-                buttonText: 'Retry'
+                title: 'Restaurant not taking orders',
+                message: 'We’re sorry, but the restaurant is not currently taking orders. Please choose another restaurant to order from.',
+                buttonText: 'Ok'
             },
 
             SERVICE_TYPE_UNAVAILABLE: {
-                title: 'We couldn\'t send your order',
-                message: 'It appears the message for this error is not yet added to the spreadsheet',
-                buttonText: 'Retry'
+                title: '{serviceType} unavailable',
+                message: 'We\'re sorry, but the restaurant is not currently taking {serviceType} orders.',
```

Comment:
If you're changing the others, this should be changed also:
```suggestion
                message: 'We’re sorry, but the restaurant is not currently taking {serviceType} orders.',
```

Reply:
Changed all of them 🙂